src/SwedbankPay/Api/Client/Version.php

```diff
@@ -52,30 +52,38 @@ protected function getVersionFromEnvironment()
         }
 
         $composerPath = __DIR__ . '/../../../../composer.json';
-        $composer = $this->readJson($composerPath);
+        if (file_exists($composerPath)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function file_exists() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5056329)